mirai-core-api/src/commonMain/kotlin/data/FriendGroup.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019-2022 Mamoe Technologies and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
+ *
+ * https://github.com/mamoe/mirai/blob/dev/LICENSE
+ */
+
+package net.mamoe.mirai.data
+
+import net.mamoe.mirai.contact.Friend
+
+public interface FriendGroup {
+    public val info: FriendGroupInfo
+
+    /**
+     * 好友分组id
+     */
+    public val id: Int
+
+    /**
+     * 好友分组名
+     */
+    public val name: String
+
+    /**
+     * 好友分组内好友数量
+     */
+    public val friendCount: Int
+
+    /**
+     * 更改好友分组名称
+     * @throws IllegalStateException 当改名不成功时
+     */
+    public suspend fun rename(newName: String)
```

Comment:
```suggestion
    public suspend fun renameTo(newName: String)
```
mirai 现有其他 API, 会修改自己对象信息的重命名为 `renameTo`, 保持一致